packages/itwinui-react/src/core/RadioTiles/RadioTile.tsx

```diff
@@ -28,33 +44,64 @@ type RadioTileProps = {
  * <RadioTile label='My tile' description='Some info' icon={<SvgSmileyHappy />} />
  */
 export const RadioTile = React.forwardRef((props, ref) => {
-  const { icon, label, description, className, style, ...rest } = props;
+  const {
+    icon,
+    label,
+    description,
+    className,
+    wrapperProps,
+    iconProps,
+    labelProps,
+    subLabelProps,
+    style,
+    ...rest
+  } = props;
 
   const inputElementRef = React.useRef<HTMLInputElement>(null);
   const refs = useMergedRefs<HTMLInputElement>(inputElementRef, ref);
 
   return (
     <Box
       as='label'
-      className={cx('iui-radio-tile', className)}
-      style={style}
       data-iui-disabled={props.disabled ? 'true' : undefined}
+      {...wrapperProps}
+      className={cx('iui-radio-tile', wrapperProps?.className)}
     >
       <Box
         as='input'
-        className='iui-radio-tile-input'
-        type='radio'
         ref={refs}
+        className={cx('iui-radio-tile-input', className)}
+        style={style}
+        type='radio'
         {...rest}
       />
       {icon && (
-        <Box as='span' className='iui-radio-tile-icon' aria-hidden>
+        <Box
+          as='span'
+          aria-hidden
+          {...iconProps}
+          className={cx('iui-radio-tile-icon', iconProps?.className)}
+        >
           {icon}
         </Box>
       )}
-      {label && <Box className='iui-radio-tile-label'>{label}</Box>}
+      {label && (
+        <Box
+          as='div'
+          {...labelProps}
+          className={cx('iui-radio-tile-label', labelProps?.className)}
+        >
+          {label}
+        </Box>
+      )}
       {description && (
-        <Box className='iui-radio-tile-sublabel'>{description}</Box>
+        <Box
+          as='div'
```

Comment:
Same fix here:

`as='div'` is the default for `<Box>`, so it doesn't need to be defined here

Reply:
done with both